src/main/java/com/mraof/minestuck/block/machine/IntellibeamLaserstationBlock.java

```diff
@@ -0,0 +1,102 @@
+package com.mraof.minestuck.block.machine;
+
+import com.mraof.minestuck.block.BlockUtil;
+import com.mraof.minestuck.block.MSBlockShapes;
+import com.mraof.minestuck.block.MSProperties;
+import com.mraof.minestuck.blockentity.IntellibeamLaserstationBlockEntity;
+import com.mraof.minestuck.blockentity.MSBlockEntityTypes;
+import com.mraof.minestuck.util.CustomVoxelShape;
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Direction;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.BlockGetter;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.EntityBlock;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.entity.BlockEntityTicker;
+import net.minecraft.world.level.block.entity.BlockEntityType;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.level.block.state.StateDefinition;
+import net.minecraft.world.level.block.state.properties.BooleanProperty;
+import net.minecraft.world.phys.BlockHitResult;
+import net.minecraft.world.phys.shapes.CollisionContext;
+import net.minecraft.world.phys.shapes.VoxelShape;
+
+import javax.annotation.Nullable;
+import java.util.Map;
+
+public class IntellibeamLaserstationBlock extends MachineBlock implements EntityBlock
```

Comment:
remove the double empty lines in this file

Reply:
done